src/controllers/v1/entities.js

```diff
@@ -57,28 +59,403 @@ module.exports = class Entities extends Abstract {
   ]
 */
 
-  /**
-   * List all the entities.
-   * @method
-   * @name list 
-   * @returns {JSON} - List of all entities.
-   */
-
-  async list(req) {
-    return new Promise(async (resolve, reject) => {
-      try {
-
-        let result = await entitiesHelper.list("all", { name: 1 })
-
-        return resolve(result)
-
-      } catch (error) {
-        return reject({
-          status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
-          message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
-          errorObject: error
-        })
-      }
-    })
-  }
-}
\ No newline at end of file
+	/**
+	 * Find all the entities.
```

Comment:
@MallanagoudaB give proper api description to all the endpoints in both entity and entityType controller, you can refer this file as an example : https://github.com/shikshalokam/ml-core-service/blob/master/controllers/v1/entities.js
(note : refer the structure and add similar description to your api end points)